ansible/templates/minigraph_link_meta.j2

```diff
@@ -52,3 +52,28 @@
     </Link>
   </LinkMetadataDeclaration>
 {% endif %}
+
+{% if autoneg_enabled is defined %}
+  <LinkMetadataDeclaration>
+    <Link xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution">
+{% for if_index in vm_topo_config['autoneg_interfaces']['intfs'] %}
+{% set autoneg_intf = "Ethernet" ~ if_index ~ "/1" %}
+        <a:LinkMetadata>
+            <a:Name i:nil="true"/>
+            <a:Properties>
+            <a:DeviceProperty>
+                 <a:Name>AutoNegotiation</a:Name>
+                 <a:Value>True</a:Value>
+            </a:DeviceProperty>
+            <a:DeviceProperty>
+                 <a:Name>FECDisabled</a:Name>
+                 <a:Reference i:nil="true"/>
+                 <a:Value>True</a:Value>
```

Comment:
@vdahiya12 disabling FEC when AN is enabled, I would suggest to use `msft_an_enabled`. Also keep `autoneg_enabled` for only enabling AN which can be used in future

Reply:
fixed